### DISCUSSION TITLE
What does the blue area of the diff in PR generally mean?

### DISCUSSION THREAD

**Discussion by songyuc** [community/community#30637] [New to GitHub]
Hi, guys,
I am new to GitHub. When I learned about [review-pull-requests](https://github.com/skills/review-pull-requests), I saw this,
![218aa0ff560689408c0eda4ac9dc57c](https://user-images.githubusercontent.com/27288110/186382213-c0a49f7e-60e8-47e7-8c5b-733d8a2bebf7.jpg)
I want to know what the blue area of the diff in PR generally means, and is there any documentation for explanation of the area color?

Your answer and guide will be appreciated!

**Comment by airtower-luna** [comment]
The part in blue describes which section of the file you's seeing, if you expand it to show more you'll see the numbers change. And if possible for the given language it shows the section of the file you're in (e.g. a function name, class name, or section header).

You'll see similar lines in diffs on the command line (e.g. from `git diff`, `git show`, or `git log`), but the coloring is likely different. :wink:
